api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -256,6 +256,8 @@ public String freeTierBillingAccountName() {
     public boolean enableFireCloudV2Billing;
     // If true, use the new rewrite version of access module.
     public boolean enableAccessModuleRewrite;
+    // If true, egress alerts are stored and automatically remediated.
```

Comment:
I don' think the alerts are automatically remediated when flag is true

Reply:
The feature is in progress, the documentation of the feature flag is meant to reflect the end-state of the feature, at which point events will be automatically remediated. I think this is generally well understood for feature flags which usually guard features that are still under development.

Is this what you meant or am I misunderstanding the comment?

Reply:
I see, the comment is for the end state. I was confused is the current state, in your code below
```
EgressEventStatus status = EgressEventStatus.REMEDIATED;
    if (workbenchConfigProvider.get().featureFlags.enableEgressAlertingV2) {
      status = EgressEventStatus.PENDING;
    }
```
if enableEgressAlertingV2 is true, alert is `PENDING`, otherwise it is `REMEDIATED`.
So at some point before release you will flip the logic in that code? When would that happen?

Reply:
I'm intending to keep the current logic:

- feature off: we store egress events, but we don't process them automatically, so we immediately store as REMEDIATED. This is because they will be remediated manually by the oncall - it is not yet the job of the system to process these. If we instead chose PENDING here, this would create a backlog of events that we'd need to deal with when we launch the feature.
- feature on: store as PENDING and file a cloud task to process (not implemented yet). Offline cron will also check for events that have been PENDING for a long time and requeue them (not implemented yet)

Reply:
Also, we don't **need** to store the events today, I could have kept this entire thing behind the feature flag, but part of the value of this table is having the historical record of events that we can query in the DB, so I'd like to collect them starting now (no behavioral changes yet).

Reply:
The code change is reasonable, I must missed some part:
1. In the comment: If true, egress alerts are stored and automatically remediated.
2. But in the code: feature on: store as PENDING and file a cloud task.

Reply:
Agree the terminology here is confusing. I tried rewriting the comment, let me know if this clarifies.

Reply:
Thanks. Now I understand remediated in the comment is not the `REMEDIATED` status in DB table.